test/config/manager/manager.yaml

```diff
@@ -44,12 +44,39 @@ spec:
       - command:
         - /root/manager
         image: controller:latest
+        imagePullPolicy: Always
         name: manager
+        env:
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: SECRET_NAME
+            value: $(WEBHOOK_SECRET_NAME)
         resources:
           limits:
             cpu: 100m
             memory: 30Mi
           requests:
             cpu: 100m
             memory: 20Mi
+        ports:
+        - containerPort: 9876
+          name: webhook-server
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /tmp/cert
+          name: cert
+          readOnly: true
       terminationGracePeriodSeconds: 10
+      volumes:
+      - name: cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-server-secret
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: webhook-server-secret
+  namespace: system
```

Comment:
Since these files are generated at project creation time, we should document somewhere in an issue steps required to migrate a project before adding a webhook.